### PR TITLE
Replaces fireball wand on lava land with Drake tail

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

The fireball wand has been replaced with a 20 burn 2 tile reach whip that stets any carbon aflame.

## Why It's Good For The Game

The fireball wand is a magic item that is has no reason to be their, it was a nod at the drakes fire attacks, but the drake itself had no magic. Replacing it with this tail whip makes more logical idea of what you "win" be defeating them rather then space magic.
It also take away, the seen to be by the upper people, a powerful weapon that has few counters outside of Chaplin. 

## Changelog
:cl:
add: The ash drag tail has been seen to be used as a powerful weapon by miners
balance: replaces the firewand from drake loot table with a whip that set ablaze and reaches.
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -825,6 +825,36 @@
 	owner.visible_message("<span class='danger'>[owner] is protected by a ring of [ghost_counter] ghosts!</span>")
 	return ..()
 
+//Fire-Tail
+
+/obj/item/melee/fire_tail
+	name = "drake tail"
+	desc = "A long flaming tail from an ash drake."
+	icon_state = "draketail"
```
Comment (Trilbyspaceclone) [inline code/modules/mining/lavaland/necropolis_chests.dm:833]: I rather like how the tail whip came out.

**Comment by ghost** [comment]
better than fireball wand but maybe without the disarm? there's already a lot of stuff going on for a 2 tile melee weapon

**Comment by Hatterhat** [comment]
belmont machine

**Comment by KeRSedChaplain** [comment]
Yeah, the disarm might make it a bit broken.

Also, I feel like chainhit.ogg doesn't fit the thing if its literally just the severed tail of an Ash drake.

**Comment by kiwedespars** [comment]
3 hit fireball wand replaced with an infinite use weapon that may or may not be better than the stunbaton.

**Comment by lolman360** [comment]
at least you have to get closer than fireball.

With fireball you land one shot across the screen and it's a knockout+damage+sets them on fire. so what if it's limited use, it literally takes one hit to guarantee a kill. if you're a non-antag you're really only going to be validhunting with this (fireball isnt effective on lavaland/vs simples). if you're an antag and you use it's a fastmoving instastun with one counter (null rod).

plus, this is at least a unique idea instead of just "hey what if murderbone antag equipment on lavaland"

**Comment by Ghommie** [comment]
More than a drake tail, that thing looks like a bullwhip, and only sets fuel-doused people aflame.
This is a pretty bland effortless brute version of the welder misguised as improvement.

**Comment by Trilbyspaceclone** [comment]
@Ghommie
The sprite came out fine, it looks good. It looks like a tail. It uses the colours of the drake.

**Comment by YakumoChen** [comment]
It maybe looks like a tail if the dragon was, in fact, a gigantic rat

but I'm pretty sure it isn't.

It looks like a recolored bullwhip

in fact I think most of this code is a word for word copy of the chain of command.

**Comment by Trilbyspaceclone** [comment]
@YakumoChen What?
The sprite stuff is coppyed from chain of comman but the code itself is just normal melee item code thats not copy'ed from CoC.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -825,6 +825,33 @@
...
+	item_state = "chain"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	damtype = "fire"
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]:
```suggestion
	damtype = BURN
```

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -825,6 +825,33 @@
...
+	force = 20
+	throwforce = 20
+	reach = 2
+	w_class = WEIGHT_CLASS_NORMAL
+	item_flags = NONE
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]: It's one awful stingy weapon, remove this line and let securibots do their job here.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 4]
```diff
@@ -825,6 +825,33 @@
...
+	icon_state = "draketail"
+	item_state = "chain"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	damtype = "fire"
+	force = 20
+	throwforce = 20
+	reach = 2
+	w_class = WEIGHT_CLASS_NORMAL
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Same value as parent type, remove.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 5]
```diff
@@ -825,6 +825,33 @@
...
+	name = "drake tail"
+	desc = "A long flaming tail from an ash drake."
+	icon_state = "draketail"
+	item_state = "chain"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	damtype = "fire"
+	force = 20
+	throwforce = 20
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Whips are not really good throwing weapons, Trilby, give it a less senseless value.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 6]
```diff
@@ -825,6 +825,33 @@
...
+//Fire-Tail
+
+/obj/item/melee/fire_tail
+	name = "drake tail"
+	desc = "A long flaming tail from an ash drake."
+	icon_state = "draketail"
+	item_state = "chain"
+	lefthand_file = 'icons/mob/inhands/weapons/melee_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/melee_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Sorry, but at this point just make this a chain of command subtype, Yakumo is not really wrong saying this is a copypasta.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 7]
```diff
@@ -825,6 +825,33 @@
...
+	force = 20
+	throwforce = 20
+	reach = 2
+	w_class = WEIGHT_CLASS_NORMAL
+	item_flags = NONE
+	total_mass = 1.5
+	attack_verb = list("flogged", "whipped", "lashed", "disciplined")
+	hitsound = 'sound/weapons/chainhit.ogg'
+
+/obj/item/melee/fire_tail/afterattack(target, mob/user, proximity_flag)
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]:
```suggestion
/obj/item/melee/fire_tail/afterattack(atom/target, mob/user, proximity_flag)
```

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 8]
```diff
@@ -825,6 +825,33 @@
...
+	. = ..()
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm:839]: add a proximity return check after this line please.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Looks like a bullwhip/rat tail, inhand sprite is the same as the command chain. Wouldn't say it's a good replacement to the fireball wand and rather keep it admin only for the time being.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 9]
```diff
@@ -825,6 +825,23 @@
...
+
+/obj/item/melee/chainofcommand/fire_tail
+	name = "drake tail"
+	desc = "A long flaming tail from an ash drake."
+	icon_state = "draketail"
+	damtype = BURN
+	force = 20
+	total_mass = 1.5
+
+/obj/item/melee/fire_tail/afterattack(atom/target, mob/user, proximity_flag)
```
Comment (Ghommie) [inline code/modules/mining/lavaland/necropolis_chests.dm]:
```suggestion
/obj/item/melee/chainofcommand/fire_tail/afterattack(atom/target, mob/user, proximity_flag)
```

**Comment by Ghommie** [comment]
Pretty janky and looks like a rat tail. No.

**Pull request closed without merging**